### PR TITLE
add scramp

### PR DESCRIPTION
**Pull request by nicolas-geniteau**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by nicolas-geniteau** [comment]
@conda-forge/help-python, ready for review. Thanks !

File: recipes/scramp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 475aa6296deb2737b86e9df9098e8eca0f30c8ad1cc0a8adadb99ef012a5ceba
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```
Comment (marcelotrevisani) [inline recipes/scramp/meta.yaml]: Could it be ``noarch``?
```suggestion
  noarch: python
```

You can select python on the requirements

File: recipes/scramp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/scramp/meta.yaml]:
```suggestion
    - python >3.5
```

File: recipes/scramp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    - pip
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/scramp/meta.yaml]:
```suggestion
    - python >3.5
```

File: recipes/scramp/meta.yaml (hunk 1 above)
Comment (marcelotrevisani) [reply recipes/scramp/meta.yaml]: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Comment (nicolas-geniteau) [reply recipes/scramp/meta.yaml]: Yes, thanks !

**Pull request merged**